client/components/credit-card-form-fields/index.jsx

```diff
@@ -145,7 +145,7 @@ export class CreditCardFormFields extends React.Component {
 			<div className="credit-card-form-fields">
 				{ this.createField( 'name', Input, {
 					autoFocus,
-					label: translate( 'Name on Card', {
+					label: translate( 'Cardholder Name', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Invalid cardholder name' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?